README.md

```diff
@@ -19,23 +20,26 @@ Each AI/ML pipeline is sorted by the composite of the name of the library, and t
 Each subdirectory is scoped to run only one AI/ML integration's suite of tests for one language within that cloned repository. For example, if an AI/ML integration has both a Python and C# implementation of Atlas Vector Search, two subdirectories need to be made: one for Python, titled `{repo}-python`, and one for C#, titled `{repo}-csharp`. See `semantic-kernel-*` subdirectories in the layout example below.
 
 Within each subdirectory you should expect to have:
-- `run.sh`  --  A script that should handle any additional library installations and steps for executing the test suite. This script should not populate the Atlas database with any required test data.
+
+- `run.sh` -- A script that should handle any additional library installations and steps for executing the test suite. This script should not populate the Atlas database with any required test data.
 - `database/` -- An optional directory used by `.evergreen/scaffold_atlas.py` to populate a MongoDB database with test data. Only provide this if your tests require pre-populated data.
 - `database/{collection}.json` -- An optional JSON file containing one or more MongoDB documents that will be uploaded to `$DATABASE.{collection}` in the local Atlas instance. Only provide this if your tests require pre-populated data.
 - `indexConfig.json` -- An optional file containing configuration for a specified Atlas Search Index.
 - Additionally, you can add other useful files, including `.env` files, if required by your tests.
 
 The general layout of this repo will looks like this:
```

Comment:
The general layout of this repo **will look** like this:

OR

The general layout of this repo **looks** like this:

Reply:
> The general layout of this repo looks like this: